formats/raw.nix

```diff
@@ -6,10 +6,13 @@
     fsType = "ext4";
   };
 
-  boot.growPartition = true;
-  boot.kernelParams = [ "console=ttyS0" ];
-  boot.loader.grub.device = lib.mkDefault "/dev/vda";
-  boot.loader.timeout = 0;
+  boot = {
+    growPartition = true;
+    kernelParams = [ "console=ttyS0" ];
+    loader.grub.device = "/dev/vda";
```

Comment:
we need the mkDefault here otherwise it crashed with the device = "nodev" from raw-efi.nix